### PR TITLE
Remove DBFilesClient.NET and use WDCReaderLib instead

### PR DESCRIPTION
**Pull request by funjoker**
Remove DBFilesClient.NET and use WDCReaderLib instead
Updated DB2 Structs from WDB6 to WDC2 (8.0.1)

Tested and works

### CODE REVIEW

File: WowPacketParser/SQL/SQLDatabase.cs [hunk 1]
```diff
@@ -104,8 +104,8 @@ private static void LoadBroadcastText()
 
                     var broadcastText = new BroadcastTextEntry()
                     {
-                        MaleText = maleText,
-                        FemaleText = femaleText,
+                        Text = maleText,
+                        Text1 = femaleText,
```
Comment (LuigiElleBalotta) [inline WowPacketParser/SQL/SQLDatabase.cs]: Why change from "FemaleText" to "Text1" ?

Comment (funjoker) [reply WowPacketParser/SQL/SQLDatabase.cs]: Original Blizzard naming

Comment (funjoker) [reply WowPacketParser/SQL/SQLDatabase.cs]: maleText and co is fixed in other PR

**Comment by funjoker** [comment]
Any reason why not merging? (after i solved the conflicts)

**Comment by Aokromes** [comment]
Conflicting files
WowPacketParser/DBC/Structures/Legion/BroadcastTextEntry.cs
WowPacketParser/SQL/SQLDatabase.cs

**Comment by funjoker** [comment]
I know

**Comment by soulfrost** [comment]
working excellent, also seems like it faster than old lib, ty for work

**Comment by funjoker** [comment]
atm WPP loads DBC twice. I just made it load them once, this speeds it up already.
The Lib itself prolly is a bit slower, but at least the Lib works with WDC1 and WDC2.

**Pull request merged**